commit edae01b94a846b39664420fe0c7561a594bb50a2
Author: Brett Rudley <brudley@broadcom.com>
Date:   Fri Oct 8 15:56:50 2010 -0700

    staging: brcm80211: Purge unused wlc_id_name
    
    Purge unused wlc_id_name_entry
    
    Signed-off-by: Brett Rudley <brudley@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/sys/wlc_mac80211.c
+++ b/drivers/staging/brcm80211/sys/wlc_mac80211.c
@@ -143,14 +143,6 @@ uint wl_msg_level =
 static wlc_info_t *wlc_info_dbg = (wlc_info_t *) (NULL);
 #endif
 
-#if defined(BCMDBG)
-struct wlc_id_name_entry {
-	int id;
-	const char *name;
-};
-typedef struct wlc_id_name_entry wlc_id_name_table_t[];
-#endif
-
 /* IOVar table */
 
 /* Parameter IDs, for use only internally to wlc -- in the wlc_iovars
